src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1299,3 +1300,27 @@ def validate(self) -> None:
 
 
 yaml.add_representer(SNMPGatewaySpec, ServiceSpec.yaml_representer)
+
+
+class MDSSpec(ServiceSpec):
+    def __init__(self,
+                 service_type: str = 'mds',
+                 service_id: Optional[str] = None,
+                 placement: Optional[PlacementSpec] = None,
+                 unmanaged: bool = False,
+                 preview_only: bool = False,
```

Comment:
@mgfritch do you think there are any other parameters we could maybe need for mds or is this enough